doxygen/qlib/Doxyfile.in

```diff
@@ -1029,11 +1035,6 @@ ENUM_VALUES_PER_LINE   = 4
 
 GENERATE_TREEVIEW      = YES
 
-# By enabling USE_INLINE_TREES, doxygen will generate the Groups, Directories,
-# and Class Hierarchy pages using a tree view instead of an ordered list.
-
-USE_INLINE_TREES       = YES
-
```

Comment:
this whole section was deleted, while the rest of the options was just commented, didn't you want to just commented out as well?